utils/wasm-instrument/src/syscalls.rs

```diff
@@ -608,7 +612,9 @@ pub enum ParamType {
     Handler,         // i32 handler number
     Alloc,           // i32 alloc pages
     Free,            // i32 free page
-    Version,         // i32 version number of exec settings
+    // i32 free upper bound for use with free_range. Should be placed after Free in fn signature
```

Comment:
the only comment above? seems strange, split into diff lines

Reply:
What about max line length guidelines? Should I just ignore them?

Reply:
What about this option?
```rust
Variant,        // comment
Variant,        // very very very long
                     // really very long comment
```

Reply:
The space before the third comment dissapears after `cargo fmt`